src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
I think we should unset the flags we don't want. The case statement will unset any flags, but then you only add in the flags that were set not unset when you `|=` the final result. If you were to initialize the temporary variable then twiddle bits and finally set it at the end you would have a better result.

```c++
c_cflags = _tio_config.c_cflag;
...
case SERIAL_8N1:
c_cflags |= CS8;
c_cflags &= ~(PARENB | PARODD | CSTOPB);
...
_tio_config.c_cflag = c_cflags;
```